src/rgw/rgw_sal_motr.cc

```diff
@@ -293,7 +293,29 @@ int MotrUser::create_bucket(const DoutPrefixProvider* dpp,
 
 int MotrUser::read_attrs(const DoutPrefixProvider* dpp, optional_yield y)
 {
-  return 0;
+  int rc = 0;
+  if (not attrs.empty())
+    return rc;
+  
+  struct MotrUserInfo muinfo;
+  bufferlist bl;
+  if (store->get_user_cache()->get(dpp, info.user_id.to_str(), bl)) {
+    // Cache miss
+    rc = store->do_idx_op_by_name(RGW_MOTR_USERS_IDX_NAME,
+                                      M0_IC_GET, info.user_id.to_str(), bl);
+    ldpp_dout(dpp, 20) << "do_idx_op_by_name() = "  << rc << dendl;
```

Comment:
Suggestion: add a printout of `__func__` here, so that it will be possible to see which function was calling `do_idx_op_by_name()` when it will fail.

Reply:
done 👍